vector-im/element-x-android
# Improve session db and improve deleted session behavior

## Type of change


## Content



## Motivation and context


Working on https://github.com/vector-im/element-android/issues/8663, I realized that some fields was missing to migrate data properly. This PR adds 2 fields to SessionData. Also adding unit test for the database migration (automatically added to the gradle `test` task).

Part of #1154 (no soft logout support)

Will add string to localazy once the wording will be approved.

## Screenshots / GIFs


See recorded screenshot

## Tests



- Sign in to an account;
- Delete the session from another device;
- Observe the new screen

## Tested devices

- OS version(s):

## Checklist
